boards/others/neorv32/Kconfig

```diff
@@ -1,5 +1,7 @@
-# Copyright (c) 2021 Henrik Brix Andersen <henrik@brixandersen.dk>
+# Copyright (c) 2021,2025 Henrik Brix Andersen <henrik@brixandersen.dk>
 # SPDX-License-Identifier: Apache-2.0
 
 config BOARD_NEORV32
-	select SOC_NEORV32_V1_8_6 if "$(BOARD_REVISION)" = "1.8.6"
+	select RISCV_ISA_RV32I
+	select RISCV_ISA_EXT_M
+	select ATOMIC_OPERATIONS_C
```

Comment:
Shouldn't `RISCV_ISA_RV32I` be selected by the SoC? I understand that the particular NEORV32 configuration that the `neorv32` platfom targets is configured as `rv32im_zicsr_zifencei`, and that NEORV32 can be e.g. configured without enabling the `M` extensions, but the RV32I part is not configurable, right?

Reply:
The RV32I part is configurable as well; it can also be synthesized as RV32E.